specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json

```diff
@@ -118,6 +118,57 @@
         }
       }
     },
+    "/providers/Microsoft.Subscription/subscriptionOperations/{operationId}": {
+      "get": {
+        "description": "Get the status of the pending Microsoft.Subscription API operations.",
+        "operationId": "SubscriptionOperation_Get",
+        "x-ms-examples": {
+          "getPendingSubscriptionOperations": {
+            "$ref": "./examples/getSubscriptionOperation.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Successful completion of the asynchronous operation",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionCreationResult"
+            }
+          },
+          "202": {
```

Comment:
GET operation should not return 202.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-operation200.md
RPC-Get-V1-01
Should return either 200 (With Value if exists) or 404 (If not exists)
Applies to get and list operations.
For list operations 404 is returned when resource group or subscription does not exist. [404 test not required for LIST operations as the behavior is in ARM and not in RP]